akka-actor-tests/src/test/java/akka/pattern/PatternsTest.java

```diff
@@ -50,12 +53,34 @@ public Receive createReceive() {
         }
     }
 
+    public static final class StopActor extends AbstractActor {
+
+        public static final class StopMessage implements NoSerializationVerificationNeeded {
+            public final TestLatch latch;
+            public final FiniteDuration duration;
+
+            public StopMessage(TestLatch latch, FiniteDuration duration) {
+                this.latch = latch;
+                this.duration = duration;
+            }
+        }
+
+        @Override
+        public Receive createReceive() {
+            return receiveBuilder()
+                    .match(StopMessage.class, message -> Await.ready(message.latch, message.duration))
```

Comment:
No need to do this blocking, just reply and verify that a quick ping-pong works with the different asks

Reply:
Get it removed, and it certainly becomes more clear.